test/run-drun/stable-region-blob.mo

```diff
@@ -0,0 +1,124 @@
+import P "mo:⛔";
+import Region "stable-region/Region";
+actor {
+
+  P.debugPrint("Begin");
+
+  let next_id = Region.next_id();
+  assert next_id == 2;
+
+  let r2 = Region.new();
+  let r2_id = Region.id r2;
+
+  P.debugPrint("Created region " # (debug_show r2_id));
+
+  assert r2_id == 2;
```

Comment:
@crusso @luc-blaeser 
When I tried using `Value.from_scalar(...)` in the implementation of this, I got `4`, not `2`.

I think it would help me to talk synchronously about how these pointer-vs-scalar conventions work.  It hasn't sunk in for me yet, clearly.  Thanks!

Reply:
Sure. In the meantime, the doc for this is in compile.ml somewhere possibly module BitTagged and Tagged IIRC